packages/gradient-block/src/components/ColorTooltip.tsx

```diff
@@ -0,0 +1,55 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { joinClassNames, toHexString } from '@frontify/guideline-blocks-shared';
+import { EditAndDeleteColorBox } from './';
+import { ColorTooltipProps } from '../types';
+import { useHover } from '@react-aria/interactions';
+
+export const ColorTooltip = ({
+    gradientColor,
+    gradientColors,
+    showColorModal,
+    setColors,
+    setShowColorModal,
+    setCurrentlyEditingPosition,
+}: ColorTooltipProps) => {
+    const left = gradientColor.position > 95 ? `calc(${gradientColor.position}% - 8px)` : `${gradientColor.position}%`;
+    const { hoverProps, isHovered } = useHover({});
+
+    const pointClasses = joinClassNames([
+        'tw-w-2 tw-h-2 tw-rounded-full -tw-mt-[22px]',
+        isHovered ? 'tw-bg-black-100' : 'tw-bg-black-20',
+    ]);
+
+    const tooltipClasses = joinClassNames([
+        'tw-inline-block tw-mt-2.5 -tw-ml-1',
+        isHovered ? 'tw-visible' : 'tw-invisible',
+    ]);
+
+    return (
+        <div className="tw-absolute tw-z-[1]" style={{ left }} {...hoverProps}>
+            <div data-test-id="color-points" className={pointClasses} />
```

Comment:
Nit: IMHO no benefit in assigning the classes to a variable first. You could also move the classes directly here.

Reply:
i thought it would be cleaner, but i adjust it because it looks also very clean if added directly